starport/pkg/protoimport/import.go

```diff
@@ -0,0 +1,49 @@
+package protoimport
```

Comment:
I don't think we need to create a separate package to import stuff in proto files tbh.  

IsImported is a good place for protonanalysis package.

I am not sure why EnsureProtoImported needs a placeholder at all. If you want to make it a generic func and place to under pkg/, I think it should not need to use placeholders. Can't you just parse the proto file, add the import programmatically and print it again?

If you don't want to make a generic EnsureProtoImported func now, you can just copy it's content to somewhere under templates/ and keep the placeholder replacement logic as it is. In the future, we'll refactor all and make proto files not depend placeholders anyway.

Reply:
make sense. I added the inside the templates. This way, we don't need to use protoanalysis pkg too much

Reply:
https://github.com/tendermint/starport/pull/1495/commits/d01657ee156bb63b72960919389e515f030ed104